setup.py

```diff
@@ -16,7 +16,7 @@ def run_tests(self):
 
 
 # also update in nsq/version.py
-version = '0.8.3'
+version = '0.8.4b1'
```

Comment:
I agree this should probably be `0.9.0-beta1`

Reply:
ok, will update ... I did the "b1" here and the "-beta1" in version.py because "b1" is more conventional for pypi, see for example https://github.com/tornadoweb/tornado/commit/7457d6f58387a801fa2c648daa341e7043734219

but I can also try "-beta1" here and see how that works for pip, probably does

Reply:
👍 i don't have a strong preference on that specific detail

Reply:
cool, I'll keep with the scheme (b1 here)
updated, will test a bit on my stuff over the next day or two